server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/utils/quotas/DefaultQuotaLoader.scala

```diff
@@ -0,0 +1,35 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.jmap.utils.quotas
+
+import javax.inject.Inject
+import org.apache.james.jmap.mail.{QuotaRoot, Quotas}
+import org.apache.james.mailbox.exception.MailboxException
+import org.apache.james.mailbox.model.{MailboxPath, QuotaRoot => JavaQuotaRoot}
+import org.apache.james.mailbox.quota.QuotaRootResolver
+
+class DefaultQuotaLoader @Inject() (quotaRootResolver: QuotaRootResolver, quotaReader: QuotaReader) extends QuotaLoader {
+  @throws[MailboxException]
+  override def getQuotas(mailboxPath: MailboxPath): Quotas = {
+    val quotaRoot: JavaQuotaRoot = quotaRootResolver.getQuotaRoot(mailboxPath)
+
+    quotaReader.retrieveQuotas(QuotaRoot.fromJava(quotaRoot))
```

Comment:
I missed that fromJava/toJava thing but it looks more a toJmap/toModel conversion, don't you think?